index.html

```diff
@@ -2376,9 +2376,10 @@ <h2 id="AudioParam">
               - T_0}{T_1 - T_0} $$
             </p>
             <p>
-              where \(V_0\) is the value at the time \(T_0\) and \(V_1\) is the
-              <code>value</code> parameter passed into this method.  It is an error if either
-              \(V_0\) or \(V_1\) is not strictly positive.
+              where \(V_0\) is the value at the time \(T_0\) and \(V_1\) is
+              the <code>value</code> parameter passed into this method.  It
+              is an error if \(V_0\) and \(V_1\) have opposite signs or if
+              either \(V_0\) or \(V_1\) is zero.
```

Comment:
Actually, I have some second thoughts about this.  I think it's useful if V_1 is zero to signal an error.  But if you're running the automation and when the exponential ramp event becomes current and we then discover that V_0 is 0, throwing an error is kind of useless.  You get notified, but there's nothing you can do about it, and the error can happen long after the exponential ramp has already finished.